src/utils/verifiable-credential/snapVC.test.ts

```diff
@@ -94,11 +99,12 @@ describe('SnapVerifiableCredential', () => {
       expect(
         vc.buildDisputedPayload(VALID_ACCOUNT_1, SNAP_SHASUM_1, ['reason']),
       ).toStrictEqual({
-        domain,
+        domain: typedDataDomain,
         message: {
           '@context': ['https://www.w3.org/2018/credentials/v2'],
           type: ['VerifiableCredential', 'StatusCredential'],
```

Comment:
Need to change `StatusCredential` to `ReviewCredential`
```suggestion
          type: ['VerifiableCredential', 'ReviewCredential'],
```